activities/releasing-the-standard/index.md

```diff
@@ -0,0 +1,28 @@
+---
+type: Index
+---
+
+# Releasing the Standard for Pubic Code
+
+## Preparing a release
+
+Start by creating an issue in the Standard for Public Code repository.
+The issue can include a target release date that we're aiming for and the new version number.
```

Comment:
Is this a 'can' (optional?) or a 'must'?

```suggestion
The issue must include the new version number and can include a target release date.
```

Reply:
It's an optional. It might also change while reviewing all the changes.